cfme/tests/containers/test_containers_services.py

```diff
@@ -38,14 +39,7 @@ def test_service_summary_rel(provider, rel):
 
     for name in ui_services:
         obj = Service(name, provider)
-        val = obj.get_detail('Properties', rel)
-
-        # the field should not be empty
-        try:
-            str(val)
-            validate_str = True
-        except ValueError:
-            pass
+        field_content = getattr(obj.summary.properties, rel).text_value
 
-        if validate_str:
-            assert len(val) != 0
+        if field_content:
+            assert len(field_content) != 0
```

Comment:
The code will never fail on this assert.

    assert getattr(obj.summary.properties, rel).text_value

is enough